BugDoom - web app/src/main/java/com/infoshareacademy/pl/model/Location.java

```diff
@@ -4,44 +4,76 @@
 import java.util.Objects;
 
 public class Location {
-    private long controlPointID;
-    private BigDecimal coordinateX;
-    private BigDecimal coordinateY;
+    private double x;
+    private double y;
+    private String text;
```

Comment:
dlaczego zmiana z BigDecimal? Po co nam ten tekst?
Widzę też, że skasowany został numer id

Reply:
BigDecimal wysypywał mi mapkę. Rozważałem też float, ale na nasze potrzeby chyba wystarczy double. Tekst miał to być np. start, meta, checkpiont/contol point

Reply:
zmienna text nie jest czytelną nazwą, nie mówi jaka jest jej funkcja. Nie rozumiem "Tekst miał to być np. start, meta, checkpiont/contol point"